.cbmc-batch/jobs/aws_cryptosdk_compare_hash_elems_by_key_string/aws_cryptosdk_compare_hash_elems_by_key_string_harness.c

```diff
@@ -19,11 +19,13 @@
 #include <proof_helpers/utils.h>
 
 struct aws_hash_element *nondet_hash_string_element_allocation(size_t max_size) {
-    struct aws_hash_element *elem = can_fail_malloc(sizeof(*elem));
+    struct aws_hash_element *elem = malloc(sizeof(*elem));
```

Comment:
Are you using `pointer-primitive-check` in these proofs?

Reply:
Yep! Just checked my local Makefile.common and it has it. I think that check was merged into the master for ESDK as well.